client/src/components/leftSidebar/topLeftLogoAndTitle.js

```diff
@@ -13,16 +14,8 @@ import Logo from "../framework/logo";
 class LeftSideBar extends React.Component {
   render() {
     const { datasetTitle, aboutURL } = this.props;
-
-    const displayTitle =
-      datasetTitle.length > globals.datasetTitleMaxCharacterCount
-        ? `${datasetTitle.substring(
-            0,
-            Math.floor(globals.datasetTitleMaxCharacterCount / 2)
-          )}…${datasetTitle.slice(
-            -Math.floor(globals.datasetTitleMaxCharacterCount / 2)
-          )}`
-        : datasetTitle;
+    const width = 190;
```

Comment:
nit: 

`WIDTH = '190px'`
`FONT_SIZE = '14px'`

Even though browser automatically appends `px` for number without explicit units, it's clearer for readers to know the exact unit being used for clarity

Also since they are constants, let's use the conventional UPPER_CASE to name the variables!

Reply:
We have a globals file for times that we need to do this, and there are long descriptive names in there so we explicitly export / import things like `export const cellxgeneTitleLeftPadding = 14;` These might fit nicely there.

Reply:
@colinmegill Should globals be holding variables that aren't "global".  It may be hurt to globalize variables and hide their values before they are accessed from outside the file.  I think file-scoped constants may have a place in our standards.